src/lib/operators/jit_operator/operators/abstract_jittable.hpp

```diff
@@ -21,6 +21,18 @@ class AbstractJittable {
 
   std::shared_ptr<AbstractJittable> next_operator() { return _next_operator; }
 
+  /**
+   * Update the operator according to the encoding and nullable information of the input table columns before
```

Comment:
How about this: “Before specialization, the Jittable is updated to include the encoding and nullability information of its input. As operators may change the nullability information, tuple_nullable_information is updated.”